clio-server/src/test/scala/org/broadinstitute/clio/server/webservice/WgsUbamWebServiceSpec.scala

```diff
@@ -133,9 +158,11 @@ class WgsUbamWebServiceSpec
     // We have to test the MemorySearchDAO because we're not going to implement
     // Elasticsearch logic in our test specs. Here, we're just verifying that
     // the web service passes the appropriate queries onto the search DAO.
-    Post("/query", Map("flowcell_barcode" -> "FC123")) ~> webService.queryall ~> check {
```

Comment:
While fixing up the `.head` scolding from Codacy I noticed this test was hitting the wrong route.

Reply:
Weird. Why didn't the test realize the URI was wrong?

Reply:
The `check` body was copy-pasted from above, so it was just re-asserting things on the input model from the first `/query` run in the test.